docs/en/setup/Configuration.md

```diff
@@ -97,6 +97,7 @@ export SW_AGENT_YourConfiguration=YourValue
 | plugin_fastapi_collect_http_params | SW_PLUGIN_FASTAPI_COLLECT_HTTP_PARAMS | <class 'bool'> | False | This config item controls that whether the FastAPI plugin should collect the parameters of the request. |
 | plugin_bottle_collect_http_params | SW_PLUGIN_BOTTLE_COLLECT_HTTP_PARAMS | <class 'bool'> | False | This config item controls that whether the Bottle plugin should collect the parameters of the request. |
 | plugin_celery_parameters_length | SW_PLUGIN_CELERY_PARAMETERS_LENGTH | <class 'int'> | 512 | The maximum length of `celery` functions parameters, longer than this will be truncated, 0 turns off |
+| plugin_grpc_ignore_method | SW_PLUGIN_GRPC_IGNORE_METHOD | <class 'str'> |  | Comma-delimited list of grpc methods to ignore |
```

Comment:
Could you share the example of this gRPC method list?

`plugin_grpc_ignore_method` should be `plugin_grpc_ignored_methods` or `plugin_grpc_ignored_method_list`